helm/microgateway/templates/secrets.yaml

```diff
@@ -0,0 +1,30 @@
+{{ if and (not (eq .Values.wso2.subscription.username "")) (not (eq .Values.wso2.subscription.password "")) }}
```

Comment:
Any special reason to add this before the license header?

Reply:
There's no specific reason for having the condition check before the file header since it does not affect the functionality of the chart. Other WSO2 products[1] follow a similar pattern, and it would be ideal to maintain consistency within all of the repositories.

[1] https://github.com/wso2/kubernetes-is/blob/5.8.x/advanced/helm/is-pattern-1/templates/secrets.yaml